src/components/Form/CreateProfileForm/ProfileData/ProfileImage.tsx

```diff
@@ -97,7 +96,7 @@ const ProfileImage = () => {
               {agreement.imageUrl ? (
                 <div className="d-flex flex-row align-items-end">
                   <img
-                    src={`${agreement.imageUrl}?${Date.now()}`}
+                    src={`${agreement.imageUrl}`}
```

Comment:
Because it caused an entire image load on any interaction with the form (it caused a bandwidth usage of several megabytes just to fill the form). If any caching invalidation need to be used, it should be based on a hash. I try to refactor it.

Reply:
Ok, I'll try to fix it

Reply:
I'll just leave a comment